src/misc.jl

```diff
@@ -4,10 +4,10 @@
 #
 # See: https://en.wikipedia.org/wiki/Multivariate_gamma_function
 #
-function logmvgamma(p::Int, a::Float64)
-    res::Float64 = p * (p - 1.0) / 4.0 * log(pi)
+function logmvgamma(p::Int, a::Real)
+    res = p * (p - 1) / 4 * log(pi * one(a))
```

Comment:
Since `/` is floating point division I think it is necessary to move the division to after the `log`.